chapters/0-introduction.tex

```diff
@@ -156,15 +156,12 @@ \section{How to read this book}
 that contain all the code, data, and meta-information needed
 for others to evaluate and reproduce your work.
 
-\begin{fullwidth}
-\begin{figure}
-	\centering
-	\includegraphics[width=1.25\linewidth]{diagrams/Intro}
-	\caption{}
-	\label{fig:intro}
-\end{figure}
-\end{fullwidth}
-
+Figure \ref{fig:intro} shows a summary of the hands-on data tasks 
+comprising  research data work.
+These are described in chapters 3 to 7.
+Chapters 1 and 2, on the other hand,
+will contextualize why these each of them are important,
```

Comment:
I do not understand this part of the sentence

Reply:
```suggestion
will contextualize why each of them are important,
```

does this help?